tests/lwip/tests/01-run.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
```

Comment:
Now that I replaced all occurrences of `pexpect.spawn()` with `pexpect.spawnu()` it does